README.dev.md

```diff
@@ -303,6 +303,7 @@ if [ $(uname) = "OpenBSD" ]; then sudo cp ./puppet/zulip/files/postgresql/zulip_
 ./tools/do-destroy-rebuild-database
 ./tools/postgres-init-test-db
 ./tools/do-destroy-rebuild-test-database
+python install_phantomjs.py
```

Comment:
Also, for consistency with the rest, this should be named `./tools/install_phantomjs` and not explicitly call python